eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SummaryStatistics.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * A Summarizer can be used to aggregate statistics for multiple primitive attributes.
+ *
+ * @since 8.1
  */
 public class SummaryStatistics<T> implements Procedure<T>
```

Comment:
I will submit a separate pull request for the serialization solution for SummaryStatistics.  I think it should continue to implement Procedure.  Separately, I found this feature request for the JDK SummaryStatistics classes which I found interesting.  (https://bugs.openjdk.java.net/browse/JDK-8043747)  I also found that Oracle Coherence had to implement their own private versions of Int/Long/DoubleSummaryStatistics, I assume because of the serialization issue.